android/src/main/java/com/reactnativecommunity/webview/RNCWebView.java

```diff
@@ -242,12 +250,35 @@ public boolean getMessagingEnabled() {
         return this.messagingEnabled;
     }
 
-    protected RNCWebViewBridge createRNCWebViewBridge(RNCWebView webView) {
-        if (bridge == null) {
-            bridge = new RNCWebViewBridge(webView);
-            addJavascriptInterface(bridge, JAVASCRIPT_INTERFACE);
+    protected void createRNCWebViewBridge(RNCWebView webView) {
+        if (getSettings().getJavaScriptEnabled() &&
+          injectedJavaScriptObject != null &&
+          !TextUtils.isEmpty(injectedJavaScriptObject)) {
+          evaluateJavascriptWithFallback("(function(){\n" +
+            "    window.ReactNativeWebView.injectedObjectJson = \"" + injectedJavaScriptObject + "\";\n" +
```

Comment:
`window.ReactNativeWebView.injectedObjectJson` should be a function returning json.
please see equivalent ios implementation:  https://github.com/react-native-webview/react-native-webview/blob/4733fe7988f1a4489f0d430ffcbe152416fe1b0e/apple/RNCWebViewImpl.m#L1692-L1695

please also test your change on the [examples/Injection](https://github.com/react-native-webview/react-native-webview/blob/master/example/examples/Injection.tsx) test case.

Reply:
updated!